lib/auth/db.go

```diff
@@ -415,6 +422,7 @@ func filterExtensions(extensions []pkix.Extension, oids ...asn1.ObjectIdentifier
 var (
 	oidExtKeyUsage    = asn1.ObjectIdentifier{2, 5, 29, 37}
 	oidSubjectAltName = asn1.ObjectIdentifier{2, 5, 29, 17}
+	oidADUserMapping  = asn1.ObjectIdentifier{1, 3, 6, 1, 4, 1, 311, 25, 2}
```

Comment:
Did you intentionally duplicate this instead of using `windows.ADUserMappingExtensionOID`?

Reply:
Yes, I wanted to keep it consistent with existing code. The whole thing is ripe for refactor: `oidSubjectAltName ` can be replaced with `windows.SubjectAltNameExtensionOID` etc. I refrained from bigger refactors in this PR to make it easier to backport, however.